# HG changeset patch
# User jwe
# Date 1199478393 0
#      Fri Jan 04 20:26:33 2008 +0000
# Branch release-3-0-x
# Node ID 113fd59c0932022d55927ded75b3bc316bec9c3e
# Parent  3093cb640697179a50d4ce85b951f5f6bc1d28b9
[3-0-0-branch @ 2008-01-04 20:26:32 by jwe]

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2008-01-04  Thomas Treichl  <Thomas.Treichl@gmx.net>
+
+	* strings/strtrim.m: Doc fix.
+
 2008-01-02  John W. Eaton  <jwe@octave.org>
 
 	* plot/print.m: Correctly handle pbm terminal.
 
 2007-12-28  John W. Eaton  <jwe@octave.org>
 
 	* miscellaneous/edit.m: Use strcat instead of fullfile to add file
 	extensions.
diff --git a/scripts/strings/strtrim.m b/scripts/strings/strtrim.m
--- a/scripts/strings/strtrim.m
+++ b/scripts/strings/strtrim.m
@@ -1,9 +1,9 @@
-## Copyright (C) 1996, 2007 Kurt Hornik
+## Copyright (C) 1996, 2007, 2008 Kurt Hornik
 ##
 ## This file is part of Octave.
 ##
 ## Octave is free software; you can redistribute it and/or modify it
 ## under the terms of the GNU General Public License as published by
 ## the Free Software Foundation; either version 3 of the License, or (at
 ## your option) any later version.
 ##
@@ -11,21 +11,23 @@
 ## WITHOUT ANY WARRANTY; without even the implied warranty of
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
-## -*- texinfo -*- @deftypefn {Function File} {} deblank (@var{s})
+## -*- texinfo -*-
+## @deftypefn {Function File} {} deblank (@var{s})
 ## Remove leading and trailing blanks and nulls from @var{s}.  If
 ## @var{s} is a matrix, @var{deblank} trims each row to the length of
 ## longest string.  If @var{s} is a cell array, operate recursively on
-## each element of the cell array. @end deftypefn
+## each element of the cell array. 
+## @end deftypefn
 
 ## Author: John Swensen <jpswensen@jhu.edu>
 
 ## This function was derived from deblank.
 
 function s = strtrim (s)
 
   if (nargin != 1)
